server/src/main/java/org/opensearch/search/profile/query/ConcurrentQueryProfileBreakdown.java

```diff
@@ -40,26 +48,32 @@ public final class ConcurrentQueryProfileBreakdown extends ContextualProfileBrea
     private long avgSliceNodeTime = 0L;
 
     // keep track of all breakdown timings per segment. package-private for testing
-    private final Map<Object, AbstractProfileBreakdown<QueryTimingType>> contexts = new ConcurrentHashMap<>();
+    private final Map<Object, ContextualProfileBreakdown> contexts = new ConcurrentHashMap<>();
```

Comment:
I think we never need `ContextualProfileBreakdown`, do we?

```suggestion
    private final Map<Object, AbstractProfileBreakdown> contexts = new ConcurrentHashMap<>();
```